react/features/toolbox/components/web/Toolbox.js

```diff
@@ -979,7 +979,7 @@ class Toolbox extends Component<Props, State> {
             <RecordButton
                 key = 'record'
                 showLabel = { true } />,
-            this._shouldShowButton('sharedvideo')
+            !this._shouldShowButton('sharedvideo')
```

Comment:
In this case while we don't have `sharedvideo` at `interface_config.js` we will see that button. But if we add `sharedvideo` it will disappear.

Reply:
Fixed